client/src/Students/StudentsForm.js

```diff
@@ -0,0 +1,62 @@
+import { useState, useEffect } from "react";
+import { useHistory, useParams} from "react-router-dom";
+import Api from "../Api";
+
+function StudentsForm() {
```

Comment:
The name of this file and component should really be StudentForm (singular Student)... this form edits ONE student, not multiple...